Лабораторная 6 уровень 1

Rean-Lyaya opened the pull request.

Rean-Lyaya committed:
Create cs

Rean-Lyaya committed:
Create cs

Rean-Lyaya committed:
Create cs

Rean-Lyaya committed:
Create CWTask1.cs

Rean-Lyaya committed:
Create CWTask2.cs

Rean-Lyaya committed:
Add files via upload

Rean-Lyaya committed:
Create  Лаба6уровень1номер5

Rean-Lyaya committed:
Create лабораторная 6 уровень 2

Rean-Lyaya committed:
Create Лабораторная 6 уровень 3

NemirovichMark commented:
Здравствуйте, Ляя.

Публичные поля - это нарушение принципов ООП
public string Name;
public int Grade;
На защите попрошу это исправить на private (или будет -2 балла).

1й номер
Используйте конструктор для передачи значений внутрь структуры
И вывод информации сделайте в виде отдельного метода.

2й номер

3й номер
ИИ ещё более ленивый, чем программисты
 // Список команд первой группы
        List<Team> groupOne = new List<Team>
        {
            new Team("Команда A1", 10),
            new Team("Команда A2", 12),
            // Добавьте команды для первой группы
        };
 // Дополните список до 12 команд для первой группы

И итоговый массив лучших команд должен быть отсортирован по убыванию баллов.

На доработку.

Rean-Lyaya committed:
Create Лабораторная 6 номер 1

Rean-Lyaya committed:
Create Лабораторная 6 номер 5 уровень 1

Rean-Lyaya committed:
Лабораторная 6 уровень 2 номер 2

Rean-Lyaya committed:
Лабораторная 6 уровень 3 номер 2

Rean-Lyaya committed:
Rename Лабораторная 6 уровень 2 номер 2 to Лабораторная 6 уровень 2  переделанный

Rean-Lyaya committed:
Rename Лабораторная 6 уровень 3 номер 2 to Лабораторная 6 уровень 3 переделанный

Rean-Lyaya committed:
Лабораторная 7 уровень 1

Rean-Lyaya committed:
Лабораторная 7 уровень 2

Rean-Lyaya committed:
Лабораторная 7 уровень 3

Rean-Lyaya committed:
Лабораторная работа 10; команда - сборная солянка

В методе Main():
Создайте массив из 3-х каталогов фильмов. Создайте массив из 20 фильмов. Добавьте в каждый каталог по 5 рандомных фильмов. Создайте массив из 20 сериалов. Добавьте в каждый каталог по одному сериалу.
Создайте объект типа MyJsonSerializer. Сохраните массив отчетов в файл raw_data.json. 
Отсортируйте фильмы по дате от старых к новым. Сохраните массив каталогов в файл data.json
Добавьте ещё по 3 сериала в каждый каталог. Отсортируйте по названию. Сохраните в файл sort_data.json.
Прочтите данные из файлов raw_data.json. data.json и sort_data.json и выведите на консоль информацию о каждом турнире. Составьте UML-диаграммы классов, процессов и состояний.

NemirovichMark commented:
Здравствуйте, Ляя.
Принял 6 и 7 лабораторные.
В 10й лабораторной код принял. А UML-диаграммы где?

Rean-Lyaya committed:
Create 2cw_20var.cs

Rean-Lyaya committed:
Create Лабораторная8.cs

NemirovichMark commented:
Здравствуйте, Ляя.
12 / 20
2 за 1й номер
1 за 2й номер
1 верная папка
1 верный путь
3 создание папки и файлов
4 сериализация и десериализация

NemirovichMark commented:
8ю лабораторную принял
10ю тоже (загрузите её себе и отправьте)
